packages/graphics/opengl-meson-aarch64/package.mk

```diff
@@ -0,0 +1,76 @@
+###############################################################################
+#      This file is part of LibreELEC - https://libreelec.tv
+#      Copyright (C) 2016-present Team LibreELEC
+#
+#  LibreELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  LibreELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with LibreELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+PKG_NAME="opengl-meson-aarch64"
+PKG_ARCH="arm aarch64"
```

Comment:
What ARCH is the mali blob? It's probably aarch64 and not compatible with arm. Do you have a blob fro arm too?

Reply:
Amlogic provides 32bit mali blobs, but I'm unsure they will work using a 64bit kernel